napari/_qt/qt_main_window.py

```diff
@@ -12,17 +12,17 @@
 app.use_app(API_NAME)
 del app
 
-from qtpy.QtWidgets import (
+from qtpy.QtWidgets import (  # noqa
     QMainWindow,
     QWidget,
     QHBoxLayout,
     QLabel,
     QAction,
     QShortcut,
 )
-from qtpy.QtGui import QKeySequence
+from qtpy.QtGui import QKeySequence  # noqa
```

Comment:
What is the "violation" here? (and in the line below)

Reply:
same as above: [E402](https://lintlyci.github.io/Flake8Rules/rules/E402.html).  the problem is the `app.use_app(API_NAME)` above it.  options are to ignore E402 globally, skip this file, or use `# noqa`

Reply:
@jni I'm guessing that it's caused by [lines 12-13](https://github.com/napari/napari/blob/master/napari/_qt/qt_main_window.py#L12-L13) coming between the `import` statements.

I don't know `napari` well enough yet (working on that, ♥️ it btw), but could:
```python
app.use_app(API_NAME)
del app
```
be moved after all the `import` statements?

Reply:
@bjlittle :tada: so great to see you here. =D If I remember correctly, it needs to come before the PyQt imports, but I'm not positive about this. The basic idea is to get VisPy and QtPy speaking the same "dialect" of Qt. Trying to figure this out for our [conda-forge recipe](https://github.com/conda-forge/staged-recipes/pull/9983), in case you might have input there! :grimacing:

Reply:
that is my memory too @jni - i don't think it can be moved